app/src/main/java/org/p2p/wallet/auth/analytics/OnBoardingAnalytics.kt

```diff
@@ -1,34 +1,32 @@
 package org.p2p.wallet.auth.analytics
 
-import org.p2p.wallet.common.analytics.Events.ONBOARD_BACKING_UP_COPYING
-import org.p2p.wallet.common.analytics.Events.ONBOARD_BACKING_UP_RENEW
-import org.p2p.wallet.common.analytics.Events.ONBOARD_BACKING_UP_SAVING
-import org.p2p.wallet.common.analytics.Events.ONBOARD_BACKUP_ERROR
-import org.p2p.wallet.common.analytics.Events.ONBOARD_BACKUP_MANUALLY
-import org.p2p.wallet.common.analytics.Events.ONBOARD_BIO_APPROVED
-import org.p2p.wallet.common.analytics.Events.ONBOARD_BIO_REJECTED
-import org.p2p.wallet.common.analytics.Events.ONBOARD_CREATE_MANUAL_INVOKED
-import org.p2p.wallet.common.analytics.Events.ONBOARD_CREATE_SEED_INVOKED
-import org.p2p.wallet.common.analytics.Events.ONBOARD_MANY_WALLETS_FOUND
-import org.p2p.wallet.common.analytics.Events.ONBOARD_NO_WALLET_FOUND
-import org.p2p.wallet.common.analytics.Events.ONBOARD_PUSH_APPROVED
-import org.p2p.wallet.common.analytics.Events.ONBOARD_PUSH_REJECTED
-import org.p2p.wallet.common.analytics.Events.ONBOARD_RESTORE_GOOGLE_INVOKED
-import org.p2p.wallet.common.analytics.Events.ONBOARD_RESTORE_MANUAL_INVOKED
-import org.p2p.wallet.common.analytics.Events.ONBOARD_SPLASH_CREATED
-import org.p2p.wallet.common.analytics.Events.ONBOARD_SPLASH_RESTORING
-import org.p2p.wallet.common.analytics.Events.ONBOARD_SPLASH_SWIPED
-import org.p2p.wallet.common.analytics.Events.ONBOARD_SPLASH_VIEWED
-import org.p2p.wallet.common.analytics.Events.ONBOARD_USERNAME_RESERVED
-import org.p2p.wallet.common.analytics.Events.ONBOARD_USERNAME_SAVED
-import org.p2p.wallet.common.analytics.Events.ONBOARD_USERNAME_SKIPPED
-import org.p2p.wallet.common.analytics.Events.ONBOARD_WALLET_CREATED
-import org.p2p.wallet.common.analytics.Events.ONBOARD_WALLET_RESTORED
-import org.p2p.wallet.common.analytics.TrackerContract
-
-class OnBoardingAnalytics(
-    private val tracker: TrackerContract
-) {
+import org.p2p.wallet.common.analytics.Analytics
+import org.p2p.wallet.common.analytics.constants.EventNames.ONBOARD_BACKING_UP_COPYING
+import org.p2p.wallet.common.analytics.constants.EventNames.ONBOARD_BACKING_UP_RENEW
+import org.p2p.wallet.common.analytics.constants.EventNames.ONBOARD_BACKING_UP_SAVING
+import org.p2p.wallet.common.analytics.constants.EventNames.ONBOARD_BACKUP_ERROR
+import org.p2p.wallet.common.analytics.constants.EventNames.ONBOARD_BACKUP_MANUALLY
+import org.p2p.wallet.common.analytics.constants.EventNames.ONBOARD_BIO_APPROVED
+import org.p2p.wallet.common.analytics.constants.EventNames.ONBOARD_BIO_REJECTED
```

Comment:
maybe we should allow wildcard imports for analytics?

Reply:
But we have autoformat in our config.xml, are you experiencing some problems with that?

Reply:
no, im just talking about that with analytics we can't experience namespace conflicts, but when using wildcard import we can reduce import code a lot!